common/components/timeline/timeline--item-alt.njk

```diff
@@ -0,0 +1,7 @@
+{{ appTimelineItemHeading(params, item) }}
+
+<div class="moj-timeline__content">
```

Comment:
What's the need for the `-alt` version of this? I wonder if it would not be a bit simpler to use the same style already used elsewhere in the MOJ Design system?

Is there a case where we would actually need to use both of these variations together? It would be good to understand the thinking behind moving of the date and whether we actually need to support both of these variations or we just consistently use one approach to position of dates.

Reply:
Zapped as part of simplification and cutting ties with original component